Payments/UserInterface/01a_ListAuthorizations.md

```diff
@@ -0,0 +1,299 @@
+
+
+
+- ![Search](../../Assets/Icons/Search.png "[Search]") (Search)   
+    Click this button to display the search bar and search for authorization transactions.   
+
+
+- ![Refresh](../../Assets/Icons/Refresh01.png "[Refresh]") (Refresh)   
+    Click this button to update the list of authorization transactions.   
+
+- ![Columns](../../Assets/Icons/Columns.png "[Columns]") Columns (x)   
+    Click this button to display the columns bar and to customize the displayed columns and the order of columns in the list. The *x* indicates the number of columns that are currently displayed in the list.   
+    > [Info] It is recommended to add the following columns to your view to get the required information and to better filter and sort the entries: Status ID, Status information (if applicable).    
+
+- ![Filter](../../Assets/Icons/Filter.png "[Filter]") Filter (x)   
+    Click this button to display the filter bar and customize the active filters. The *x* indicates the number of filters that are currently active.  
+
+- [x]     
+    Select the checkbox to the left of one list entry to display the editing toolbar for this single entry.   
+    If you click the checkbox in the header, all entries in the list are selected. The editing toolbar shows all actions you can perform on several entries.
+
+
+- [CAPTURE]   
+   Click this button to capture the transaction manually, so that the order process can be continued. For detailed information, see [Capture authorization transaction](../Operation/01_ManageAuthorizations.md#capture-authorization-transaction).
+
+- [CANCEL]   
+   Click this button to cancel the transaction manually, so that the order is cancelled both at Actindo and at the payment service provider. For detailed information, see [Cancel authorization transaction](../Operation/01_ManageAuthorizations.md#cancel-authorization-transaction).
+
+- [VOID]   
+  Click this button to disable the transaction for the Actindo database. For detailed information, see [Void authorization transaction](../Operation/01_ManageAuthorizations.md#void-authorization-transaction).
+
+
+The list displays all authorization transactions. Depending on the payment service provider, the displayed columns vary. All fields are read-only. The following standard columns are available:
+
+- *External ID*   
+   ID given by the payment service provider.
+
+- *Original reference*   
+   Reference to the authorization transaction. You can use this reference to follow up one payment over the whole process.
+
+- *Transaction date*   
+   Date on which the transaction has been created by the payment service provider.
+
+- *Status*   
+   Status text.   
+
+- *Status ID*   
+   Identifier for the status. The following statuses are available, each prefixed by the ID in the list below. You can use the *Status ID* to filter and sort the list. 
```

Comment:
Ich würde Status und Status ID separat halten, aber nacheinander beschreiben, wie hier und in Operation, so ist es auch deutlich. Genau genommen, ist Status Preparing, Unconfirmed usw. und nicht 1 - Preparing, 2 - Unconfirmed... So wird es auch in der jeweiligen Spalten angezeigt.

Reply:
Man sollte nicht zwei mal untereinander die gleiche Information schreiben